pkg/distro/rhel7/distro_test.go

```diff
@@ -445,7 +445,7 @@ func TestDistroFactory(t *testing.T) {
 			expected: rhel7.DistroFactory("rhel-7.9"),
 		},
 		{
-			strID:    "fedora-37",
+			strID:    "fedora-38",
```

Comment:
There's a very little value in making these changes going forward, but it does not hurt for the sake of consistency... :slightly_smiling_face: